### ISSUE TITLE
There is no extension tab

### ISSUE THREAD

**Issue by kakazex** [miaoshouai/miaoshouai-assistant#33]
There is no extension tab, reinstalled several times, restarted everything, useless.

**Comment by salco** [comment]
for me I got an error in the console telling me there is an issue here:  
 https://github.com/miaoshouai/miaoshouai-assistant/blob/main/scripts/runtime/msai_prelude.py#L116

```
  TypeError: 'type' object is not subscriptable
```

**Comment by salco** [comment]
@kakazex Em dont know if it the same thing for you but for my linux I was using Python 3.8 and now that I've fixed it with 3.10 this extension work as expected. So check your version of python to be sure it's not just that

**Comment by kakazex** [comment]
python 3.10.6, according to the requirements. Win10 64.

**Comment by salco** [comment]
@kakazex can you try something, go under the extention folder `stable-diffusion/extensions/miaoshouai-assistant/configs` and delete the `civitai_models.json` file. re-launch stable-diffusion and tell me if you see any error related to `noneType`. I dont know if it's only that but for me it solve my main issue.

**Comment by magejosh** [comment]
Yeah, so it worked at first. Had a memory error due to multitasking (windows 10) and crashed webui. Restarted system and now extension tab is gone. tried deleting that civitai_models.json and restarting but got no love there. still no tab. Deleted extension after disabling it within the webui. reinstalled after restarting webui, still no tab. disabled it and deleted again. also deleted venv folder for webui. started webui again, installed the extension again once it was up and running then restarted the webui again after install, still no tab. 
Yet, i do see console notes about memory release from the assistant after a generation.

**Comment by miaoshouai** [comment]
what is the error message?

> Yeah, so it worked at first. Had a memory error due to multitasking (windows 10) and crashed webui. Restarted system and now extension tab is gone. tried deleting that civitai_models.json and restarting but got no love there. still no tab. Deleted extension after disabling it within the webui. reinstalled after restarting webui, still no tab. disabled it and deleted again. also deleted venv folder for webui. started webui again, installed the extension again once it was up and running then restarted the webui again after install, still no tab. Yet, i do see console notes about memory release from the assistant after a generation.

**Comment by cashea** [comment]
No tab here either.  Unfortunately, all the miaoshouai-assistant logs are empty.  Also, I do not have a civitai_models.json within the configs folder to delete.

**Comment by miaoshouai** [comment]
updated the version to 1.7, seems fixed some of the tab missing problem. If your files are not downloaded in assets folder, try to download the files in the release folder and unzip that to extensions folder under auto1111.

**Comment by triphop** [comment]
does the issue still exist?

**Comment by cashea** [comment]
I switched to Pinokio to install Automatic1111, but that isn't panning out very well.  I haven't had a chance to get back to trying m-a.  Thanks for the followup though.

**Comment by cashea** [comment]
I started the install and left it to run while I slept.  The install took.  Some assets source from a very slow connection.

**Comment by magejosh** [comment]
> what is the error message?
> 
> > Yeah, so it worked at first. Had a memory error due to multitasking (windows 10) and crashed webui. Restarted system and now extension tab is gone. tried deleting that civitai_models.json and restarting but got no love there. still no tab. Deleted extension after disabling it within the webui. reinstalled after restarting webui, still no tab. disabled it and deleted again. also deleted venv folder for webui. started webui again, installed the extension again once it was up and running then restarted the webui again after install, still no tab. Yet, i do see console notes about memory release from the assistant after a generation.

Oops, wound up fixing it. Fairly sure it was a dependency conflict. Pretty sure deleting venv and reinstalling is what fixed it for me. Sorry, will try to do better about reporting back more timely next time.
